src/booleans.js

```diff
@@ -1,59 +1,123 @@
 export const negate = (a) => {
-  // your code here
+  return a === false;
 };
 
 export const both = (a, b) => {
-  // your code here
+  if (a && b === true){
+    return true;
+  }else{
+    return false;
+  }
 };
 
 export const either = (a, b) => {
-  // your code here
+  if (a || b === true){
+    return true;
+  }else{
+    return false;
+  }
```

Comment:
`if bla then return true, else return false` is usually a sign you can get away with a one line return.